files/zh-cn/web/accessibility/keyboard-navigable_javascript_widgets/index.md

```diff
@@ -1,105 +1,78 @@
 ---
 title: 键盘导航的 JavaScript 组件
 slug: Web/Accessibility/Keyboard-navigable_JavaScript_widgets
+l10n:
+  sourceCommit: f6d04a43eadf5ab26a3488942dfb318b58234eb5
 ---
 
 <section id="Quick_links">
-  {{ListSubpagesForSidebar("Web/Accessibility", 1)}}
+  {{ListSubpagesForSidebar("/zh-CN/docs/Web/Accessibility", 1)}}
```

Comment:
Hey @jasonren0403. I wonder if we could fix the problem of `ListSubpagesForSidebar` (only displaying en-US titles) so we could list all the articles in the sidebar. There are only a few of them listed (aka, which is translated into Chinese)

Reply:
Try to fix this in mdn/yari#8241. Let's revert the change here